onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -529,14 +531,15 @@ public void setOnFocusChangeListener(OnFocusChangedListener onFocusChangedListen
     final Handler mStopChangedHandler = new Handler();
 
     public void onFocusChanged(final ObaStop stop, final HashMap<String, ObaRoute> routes,
-            final Location location) {
+                               final Location location) {
         // Run in a separate thread, to avoid blocking UI for long running events
         mStopChangedHandler.post(new Runnable() {
             public void run() {
                 if (stop != null) {
                     mFocusStopId = stop.getId();
                     //Log.d(TAG, "Focused changed to " + stop.getName());
                 } else {
+                    mFocusStopId = null;
```

Comment:
@cagryInside Could you please take a look at this PR as well?  I'm about 99.9% sure that this line `mFocusStopId = null` should be here when the stop loses focus - I believe I somehow missed this in the initial implementation of `BaseMapFragment`.  i'd like you to take a look and confirm that adding the assignment to `null` isn't going to cause any problems for any of the activities using this fragment, including the issue reporting activities, as well as the `BaseMapFragment` internally.  Specifically, the `mFocusStopId` is written to the `Bundle` in `onSaveInstanceState()` - so previous to this patch, the previous String stopId would always have been saved to the bundle (e.g., on orientation change).  Now, if the stop lost focus, that value will be `null`.  Inside `BaseMapFragment`, it doesn't appear as though `mFocusStopId` has been used - this patch is the first code to use it, AFAIK (which is probably why I missed the initial null assignment - it wasn't actually being used).  I looked at HomeActivity and I don't believe there should be any adverse affects there.  Could you give this a second look, and take a look at your use of `BaseMapFragment` with this patch in issue reporting as well?